prime-router/metadata/HL7/catchall/hl7/segments/IN1/Coverage.yml

```diff
@@ -0,0 +1,16 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
```

Comment:
Was going to do bare bones Coverage integration tests on my task for CPExtension, will move those here now so they are present from day one. A basic IN1 segment can't be added to OML_O21 integration tests yet because those won't exist in main until after OML-Updates-Base is merged in.